packages/cluster/src/commands/list.js

```diff
@@ -45,20 +47,22 @@ const run = async function (argv) {
             'DHIS2 Version',
             'DB Version',
             'Status',
-        ],
+        ].concat(anyCustomContext ? 'Context Path' : []),
     })
 
     await Promise.all(
         clusters.map(async cluster => {
             const status = await getStatus(cluster)
-            table.push([
-                chalk.blue(cluster.name),
-                cluster.port,
-                cluster.channel,
-                cluster.dhis2Version,
-                cluster.dbVersion,
-                formatStatus(status),
-            ])
+            table.push(
+                [
+                    chalk.blue(cluster.name),
+                    cluster.port,
+                    cluster.channel,
+                    cluster.dhis2Version,
+                    cluster.dbVersion,
+                    formatStatus(status),
+                ].concat(anyCustomContext ? cluster.contextPath : [])
```

Comment:
Maybe `cluster.contextPath || '/'` ?

Reply:
If you think we should show this regardless, I agree, we should probably do that.